src/QuartoGameState.java

```diff
@@ -103,36 +103,19 @@ public int calcNodesInGeneration(int generations) {
      * @param limboPiece Piece to be placed at current state
      */
     public void evaluate(QuartoPiece limboPiece) {
-        System.out.println("New evaluate\n");
         if(this.hasQuarto()) {
-            value = isMaxState ? 27 : -27;
-            System.out.println("Found Quarto State: " + value);
-            this.board.printBoardState();
-            System.out.println("\n");
+            value = isMaxState ? -27 : 27;
         }
         else {
-            System.out.println("\nStarting board state");
-            this.board.printBoardState();
-            System.out.println("\n");
-
-            // FIXME/OPTIMIZE
```

Comment:
Should still be optimized, or at least de-hackyfied. Having a second hasQuarto method is pretty hacky.